lib/lint_roller/plugin.rb

```diff
@@ -18,5 +18,15 @@ def supported?(context)
     def rules(context)
       raise Error.new("Please implement `rules(context)` and return an instance of LintRoller::Rules")
     end
+
+    def to_s
+      if about.name && about.version
+        "#{about.name} #{about.version}"
+      elsif about.name
```

Comment:
I'm wondering if the version should be optional, because even for built-in plugins you still have the version of the containing library. But I guess some people will have RuboCop or Standard plugins in their Rails apps as well. Kind of weird for me, but I've seen this in wild...

Reply:
Yeah, that's mostly the reason. More practically, this implementation is based on the fact that the attributes in `About` are not required. Ideally, both the "name and version" should be present, but when they are not, the decision is to display only the "name" instead of the result of `inspect`.